### DISCUSSION TITLE
what's RotationsPerSecond meaning?

### DISCUSSION THREAD

**Discussion by chongjeelee** [microsoft/AirSim#4234] [General]
in the settings.json file, the lidar setting parameter RotationPerSecond mean one circle per second,or one degree per second?my  opinion is one circle per second ,not one degree per second,right?

**Comment by zimmy87** [comment]
Yes, you would be correct. If you look https://github.com/microsoft/AirSim/blob/master/Unreal/Plugins/AirSim/Source/UnrealSensors/UnrealLidarSensor.cpp#L69 you see the RotationsPerSecond parameter get multiplied by 360 to get an angle distance, which would mean this parameter indicates number of full rotations/circles per second.
